examples/navier-stokes/README.md

```diff
@@ -92,7 +92,9 @@ Energy Density:
 ## Density Current
 
 This problem solves the full compressible Navier-Stokes equations, using
-operator composition and design of coupled solvers in the context of atmospheric modeling.
+operator composition and design of coupled solvers in the context of atmospheric modeling. This problem was uses the formulation given in Semi-Implicit Formulations
```

Comment:
Thanks for adding the reference paper. Remove "was".

Reply:
Good catch. Fixed.